topic/graphql/README.md

```diff
@@ -0,0 +1,71 @@
+# Example Flask Graphene-SQLAlchemy project showing GraphQL against CrateDB
+
+This example project derived from the
+[graphene-sqlalchemy flask_sqlalchemy example](https://github.com/graphql-python/graphene-sqlalchemy/tree/master/examples/flask_sqlalchemy)
+demos how GraphQL queries can be run against data stored in CrateDB using
+Graphene-SQLAlchemy. This way, the data is accessible both via SQL and GraphQL.
+
+To run this example, you will need an instance of CrateDB. You can get an
+instance of
+[CrateDB running locally with docker](https://cratedb.com/docs/guide/install/container/index.html#install-container)
+
+The project contains two models, one named `Department` and another named
+`Employee`.
+
+## Getting started
+
+First, it is a good idea (but not required) to create a virtual environment for
+this project. We'll do this using
+[venv](https://docs.python.org/3/library/venv.html) to keep things simple:
+
+```bash
+python -m venv ./graphql-demo
+source ./graphql-demo/bin/activate
+```
+
+Now we will need to get the source of the project. Do this by cloning the whole
+cratedb-examples repository:
+
+```bash
+# Get the example project code
+git clone https://github.com/crate/cratedb-examples.git
+cd cratedb-examples/topic/graphql
+```
+
+Now we can install our dependencies:
+
+```bash
+pip install sqlalchemy-cratedb graphene_sqlalchemy Flask Flask-GraphQL
```

Comment:
Just a typo-fix on the package name. Both variants work, but this one is canonical.
```suggestion
pip install sqlalchemy-cratedb graphene-sqlalchemy Flask Flask-GraphQL
```